### PR TITLE
Upgrade subnet-evm to v0.5.11

### PR DESCRIPTION
**Pull request by cam-schultz**
## Why this should be merged
Upgrades subnet-evm to v0.5.11 and avalanchego to v1.10.18. Unblocks https://github.com/ava-labs/teleporter/issues/263

## How this works
Self explanatory for the most part. One change to note is that the relayer in the E2E test is now configured with NetworkID=0, rather than NetworkID=1337. This is to workaround a bug in avalanche-network-runner that has since been [merged](https://github.com/ava-labs/avalanche-network-runner/pull/697), but not yet included in a release.

## How this was tested
CI

## How is this documented
N/A

### CODE REVIEW

**Review by michaelkaplan13 — APPROVED** [review]
🚀

File: peers/external_handler.go [hunk 1]
```diff
@@ -125,6 +126,7 @@ func (h *RelayerExternalHandler) HandleInbound(_ context.Context, inboundMessage
 			h.responseChans[blockchainID] <- inboundMessage
 		}(inboundMessage, blockchainID)
 	} else {
+		h.log.Debug("ignoring message", zap.Stringer("op", inboundMessage.Op()))
```
Comment (minghinmatthewlam) [inline peers/external_handler.go:129]: nit: capitalize log

File: tests/basic_relay.go [hunk 2]
```diff
@@ -96,8 +96,9 @@ func BasicRelay(network interfaces.LocalNetwork) {
 	)
 
 	relayerConfig := config.Config{
-		LogLevel:          logging.Info.LowerString(),
-		NetworkID:         peers.LocalNetworkID,
+		LogLevel: logging.Debug.LowerString(),
```
Comment (minghinmatthewlam) [inline tests/basic_relay.go:99]: do we want to leave as debug?

File: tests/basic_relay.go [hunk 3]
```diff
@@ -96,8 +96,9 @@ func BasicRelay(network interfaces.LocalNetwork) {
...
+		// TODO: There's currently a bug in ANR v1.7.4-rc.0 that specifies the network ID as 0. We should change this back to peers.LocalNetworkID once fixed.
+		NetworkID:         0,
```
Comment (minghinmatthewlam) [inline tests/basic_relay.go:101]: can you update `peer.LocalNetworkID` to use `github.com/ava-labs/avalanche-network-runner/utils/constants.DefaultNetworkID`?

**Review by geoff-vball — DISMISSED** [review]

File: tests/basic_relay.go (hunk 2 above)
Comment (cam-schultz) [reply tests/basic_relay.go]: Whoops, reverted

**Review by geoff-vball — APPROVED** [review]

**Review by minghinmatthewlam — APPROVED** [review]

**Review by michaelkaplan13 — APPROVED** [review]

**Pull request merged**